### ISSUE TITLE
How many epochs？

### ISSUE THREAD

**Issue by D222097** [intuitive-robots/flower_vla_calvin#15]
Could you clarify how many epochs correspond to 300k-400k training steps in the paper? Thank you for taking the time to respond!

**Comment by mbreuss** [comment]
Hi,

thanks for the questions, honestly I don't know since my pretraining code is not considering the number of epochs and only logging the number of steps.

**Comment by D222097** [comment]
Hi, @mbreuss  I have reopened this issue to seek clarification on How to tell if training is going well? 

Loss only, or other metrics we should track? Do we just need to run the downstream SFT task for validation?

**Comment by mbreuss** [comment]
the issue with flow-based policies is that the val loss does not correlate with the success rate of downstream tasks. I would recommend to include some sim data in the mix and test policies every 50k/100k steps to get a feeling for the training. Sorry that this is not very scientific and more of a vibe thing
